[2024-02-26T17:12:49.523+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exchange_trade_data.write_to_motherduck_from_data_frame manual__2024-02-26T17:12:35.169370+00:00 [queued]>
[2024-02-26T17:12:49.549+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exchange_trade_data.write_to_motherduck_from_data_frame manual__2024-02-26T17:12:35.169370+00:00 [queued]>
[2024-02-26T17:12:49.551+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-26T17:12:49.610+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): write_to_motherduck_from_data_frame> on 2024-02-26 17:12:35.169370+00:00
[2024-02-26T17:12:49.624+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'exchange_trade_data', 'write_to_motherduck_from_data_frame', 'manual__2024-02-26T17:12:35.169370+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/exchange.py', '--cfg-path', '/tmp/tmp9_qiecrj']
[2024-02-26T17:12:49.629+0000] {standard_task_runner.py:88} INFO - Job 32: Subtask write_to_motherduck_from_data_frame
[2024-02-26T17:12:49.617+0000] {standard_task_runner.py:60} INFO - Started process 7150 to run task
[2024-02-26T17:12:49.690+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-02-26T17:12:49.769+0000] {task_command.py:423} INFO - Running <TaskInstance: exchange_trade_data.write_to_motherduck_from_data_frame manual__2024-02-26T17:12:35.169370+00:00 [running]> on host e16193345271
[2024-02-26T17:12:50.366+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='exchange_trade_data' AIRFLOW_CTX_TASK_ID='write_to_motherduck_from_data_frame' AIRFLOW_CTX_EXECUTION_DATE='2024-02-26T17:12:35.169370+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-26T17:12:35.169370+00:00'
[2024-02-26T17:12:57.487+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 241, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/exchange.py", line 84, in write_to_motherduck_from_data_frame
    con = db.connect(f'md:{database_name}?motherduck_token={motherduck_token}')
duckdb.duckdb.InvalidInputException: Invalid Input Error: Initialization function "motherduck_init" from file "/home/***/.duckdb/extensions/v0.9.2/linux_amd64_gcc4/motherduck.duckdb_extension" threw an exception: "Invalid Input Error: Initialization function "motherduck_init" from file "/home/***/.duckdb/extensions/v0.9.2/linux_amd64_gcc4/motherduck.v1.15.17.duckdb_extension" threw an exception: "IO Error: RPC 'GET_WELCOME_PACK' failed: Jwt is not in the form of Header.Payload.Signature with two dots and 3 sections (UNAUTHENTICATED, request id: '818de667-c6c4-42ae-9948-3d1c8bc04797')""
[2024-02-26T17:12:57.508+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=exchange_trade_data, task_id=write_to_motherduck_from_data_frame, execution_date=20240226T171235, start_date=20240226T171249, end_date=20240226T171257
[2024-02-26T17:12:57.570+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 32 for task write_to_motherduck_from_data_frame (Invalid Input Error: Initialization function "motherduck_init" from file "/home/***/.duckdb/extensions/v0.9.2/linux_amd64_gcc4/motherduck.duckdb_extension" threw an exception: "Invalid Input Error: Initialization function "motherduck_init" from file "/home/***/.duckdb/extensions/v0.9.2/linux_amd64_gcc4/motherduck.v1.15.17.duckdb_extension" threw an exception: "IO Error: RPC 'GET_WELCOME_PACK' failed: Jwt is not in the form of Header.Payload.Signature with two dots and 3 sections (UNAUTHENTICATED, request id: '818de667-c6c4-42ae-9948-3d1c8bc04797')""; 7150)
[2024-02-26T17:12:57.653+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-26T17:12:58.096+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
